docs/build-&-deployment.md

```diff
@@ -194,15 +223,27 @@ Go to the root folder of the project in the terminal. Install all the dependenci
 
 #### Step 5:
 
-**Build an application:**
-
-Run `cd android && ./gradlew :app:assembleMosipDebug && cd ..` to build the application.
-
 **Build and run the application on the device:**
 
 * Run `npm run android:mosip` to build and install the application on the device.
 * Run `npm run android:mosip --reset-cache` to build and install the application if any change is made in the .env file.
 
+If you encounter the below issue on Windows
+```
+FAILURE: Build failed with an exception.
+
+- Where:
+  Script 'C:\....\inji\node_modules\expo\scripts\autolinking.gradle' line: 2
+
+- What went wrong:
+  A problem occurred evaluating script.
+  > Could not read script 'C:\"PATH"\inji\node_modules\expo\scripts\android\autolinking_implementation.gradle' as it does not exist.
+```
+Run this `npm i expo-modules-autolinking@~1.1.0` and rebuild the app
+
+* Path for debug apk in Inji directory
```

Comment:
Path for debug apk in Inji directory `android/app/build/outputs/apk/mosip/debug`